### DISCUSSION TITLE
Stackblitz

### DISCUSSION THREAD

**Discussion by NoelDeMartin** [vuejs/docs#1516] [General]
I've been looking at the new documentation, and I was surprised when the first link to play around with Vue didn't work with Firefox (and apparently, doesn't work with Safari either). I didn't gave it much thought and moved on.

However, later on, I had to report an issue in core and faced stackblitz again. This time, it was more annoying because I don't use Chrome on a regular basis so I had to log into github, 2FA and all, in order to fork the stackblitz template and reproduce the bug.

I don't know if this has been discussed elsewhere, but I couldn't find anything after a quick search. Why was stackblitz chosen as the main sandbox environment for Vue? Personally, I don't use this type of websites at all, so I can't tell why it's better or worse than others. But at the very least, the few ones I've tried like [codesandbox](https://codesandbox.io/) and [codepen](https://codepen.io) seem to work in Firefox. Given that this is part of the "Quick Start" guide, I'd say it's pretty important that it works in all browsers. Or at the very least, in all browsers that are supported by Vue itself (I guess it's ok if it doesn't work in IE :sweat_smile:).

**Comment by tony19** [comment]
StackBlitz has since been updated to work in Firefox.
